### DISCUSSION TITLE
Quick start button(s)

### DISCUSSION THREAD

**Discussion by digitaldirk** [linkdotnet/Blog#388] [Ideas]
Would be nice to have Deploy to X buttons on the readme. 
 
**Azure**
Docs: https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/deploy-to-azure-button

Example usage:
https://github.com/ADefWebserver/Blazor-Blogs

Seems like AWS and other may also have this ability.

Shoutout to @ADefWebserver for introducing me to this :)

**Comment by linkdotnet** [comment]
I like that - not in the main README.md - but we can add a chapter to guide people in the deployment process!
